tests/unit/sagemaker/test_deserializers.py

```diff
@@ -24,7 +25,14 @@
     StreamDeserializer,
     NumpyDeserializer,
     JSONDeserializer,
+    PandasDeserializer,
 )
+from sagemaker.utils import DeferredError
+
+try:
+    import pandas as pd
+except ImportError as e:
+    pd = DeferredError(e)
```

Comment:
I'm not sure if I should be using DeferredError in the unit tests.

Reply:
no need to in the tests